architecture/core_concepts/routes.adoc

```diff
@@ -37,6 +43,29 @@ Configuration] guide for information on deploying a router.
 ====
 endif::[]
 
+A router uses the service selector to find the service and the endpoints backing
+the service. Service-provided load balancing is bypassed and replaced with the
+router's own load balancing. Routers watch the cluster API and automatically
+update their own configuration according to any relevant changes in the API
```

Comment:
I'd replace "in the API objects" with "to the IP addresses of pods that provide the service"